### PR TITLE
docs: automation to publish documentation after a release.

### PR DESCRIPTION
**Pull request by winder**
## Summary

Two parts of this PR:
1. Minor changes to align with [the docs repo](https://github.com/algorand/docs/pull/1048).
2. Automation to push changes to the docs repo.

## Automation
Implemented in a new `Doc Repo PR Generator / docs-pr` workflow. It is designed to run manually, and to run after the `goreleaser` action.

The following are updated:
* CLI documentation, this is already automated by `reformat.py` in the docs repo.
* Copy the manually written docs from Conduit's `docs` directory into the docs repo `docs/get-details/conduit` directory.

A PR is created on the `algorand/docs` repo with a number of reviewers tagged automatically. I grabbed the list from other GH actions, maybe a group could be created in the future.

### remote repository / security notes

In order to create a PR on a remote repo, some special access is required. [I followed the steps documented here](https://github.com/peter-evans/create-pull-request/blob/main/docs/concepts-guidelines.md#creating-pull-requests-in-a-remote-repository) and created a Personal Access Token (PAT). ~The token is a classic token, because I was not able to create a fine-grained token targeting repositories in the `algorand` org.~ The token is a fine-grained token limited to the `algorand/docs` repo.

## Test Plan

In order to gain access to the `${{ secrets.PAT }}` I temporarily changed the `on` condition to `pull_request`, and pushed a branch directly to `algorand/conduit`. After getting the action to run I see the PR created:
https://github.com/algorand/docs/pull/1064

## Open Issue

The PR seems to be created by me. I tried to override the committer but I must have missed something. Also the CLA Assistant is not working. It looks like the commit is correctly changed to "algo-devops-service" but other parts are not. Maybe it's due to the personal access token?

### CODE REVIEW

File: .github/workflows/docs-pr.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        uses: peter-evans/create-pull-request@v4
+        with:
+          path: docs
+          branch: "conduit-automated-pr"
+          title: "Conduit Documentation Update"
+          body: "Regenerate CLI documentation and update hand written documentation."
+          reviewers: "nullun,barnjamin,ryanrfox"
+          delete-branch: true
+          base: staging
+          author: algo-devops-service <ci-bot@algorand.com>
```
[the remark this answers is not in the document]
Comment (winder) [reply .github/workflows/docs-pr.yml:53]: Is it the same email address, like this?
```suggestion
          author: algo-dev-service <ci-bot@algorand.com>
```

**Review by algobarb — APPROVED** [review]
I think testing it once more might be a good idea, but the github workflow LGTM.

**Comment by tzaffi** [comment]
> Maybe it's due to the personal access token?

Have you tried using the newer [Fine Grained PAT](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#fine-grained-personal-access-tokens) bs. the "Classic" one?

**Comment by winder** [comment]
> > Maybe it's due to the personal access token?
> 
> Have you tried using the newer [Fine Grained PAT](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#fine-grained-personal-access-tokens) vs. the "Classic" one?

Yes, John & Barbara are helping out with that. Looks like the fine grained token is going to work nicely.

**Pull request merged**